### DISCUSSION TITLE
Doesn't work on QFUS Y62B8G

### DISCUSSION THREAD

**Discussion by calcioscacchi** [roleoroleo/yi-hack-Allwinner-v2#734] [Q&A] [answered]
Hi there, 

i've just buy a YI 1080p home camera, paired with the app and update the firmware to 12.0.35.8_202211241029.

I download the y291ga pack, unzip it in my SD root (microSD is 2GB sized). Then I restart my cam.

The yellow light does not flashing. It starts with a still light, the blue light flashes for a few seconds and we have static blue light in the end.

I try to connect to my cam IP but it's not reacheable... Suggestions? Thanks.

**Comment by roleoroleo** [comment]
Check if a folder Factory.done is present.
This folder means that the hack is properly installed.

**Reply by calcioscacchi** [reply, marked as answer]
The problem was the SD Card: I tried with 16GB Card and it works fine ;)
